tests/integration/syscalls_tests.rs

```diff
@@ -188,3 +190,61 @@ async fn test_syscall_get_block_timestamp_cairo0(
     .await
     .expect("OS run failed");
 }
+
+#[rstest]
+// We need to use the multi_thread runtime to use task::block_in_place for sync -> async calls.
+#[tokio::test(flavor = "multi_thread", worker_threads = 1)]
+async fn test_syscall_get_tx_info_cairo0(
+    #[future] initial_state_cairo1: StarknetTestState,
+    block_context: BlockContext,
+    max_fee: Fee,
+) {
+    let initial_state = initial_state_cairo1.await;
+
+    let sender_address = initial_state.cairo1_contracts.get("account_with_dummy_validate").unwrap().address;
+    let contract_address = initial_state.cairo0_contracts.get("test_contract").unwrap().address;
+
+    let tx_version = TransactionVersion::ZERO;
+
+    let expected_chain_id = stark_felt!(1536727068981429685321u128);
```

Comment:
Is this the same as `pub const SN_GOERLI: &str = "534e5f474f45524c49";` defined in `src/config.rs`? It would be good to reuse a constant if possible or at least explain where this comes from

Reply:
It's not the same (I think this is Sepolia) but we can use Goerli and reuse the constant :+1: